Automattic/wp-calypso
# Comments: move into blocks

In #7423, @fraying requested a bunch of improvements to full post comments.

As groundwork for this, I've duplicated the `reader/comments` components into the new `blocks` directory and resolved any linting/style/CSS errors along the way.

### To test

Check that there are no functional or visual regressions in the comments section on full post view. 

The new comments components are enabled in the new Reader full post view (only in use on development and calypso.live).

Existing:
https://wpcalypso.wordpress.com/read/feeds/40474296/posts/999143144

Shiny new:
http://calypso.localhost:3000/read/feeds/40474296/posts/999143144

Test live: https://calypso.live/?branch=update/reader/comment-layout
